packages/common/src/config/db.ts

```diff
@@ -0,0 +1,33 @@
+import { drizzle, NodePgDatabase } from 'drizzle-orm/node-postgres';
+import pg from 'pg';
+import { dbInfo } from './config.js';
+
+export class DB {
```

Comment:
we could implement this as a curry function, with an initialiser first layer, that creates and returns an instance to be used where needed. Seems counterintuitive to start using classes here when we're using functional composition everywhere else in the app